critiquebrainz/frontend/views/moderators.py

```diff
@@ -0,0 +1,11 @@
+from flask import Blueprint, render_template
+from critiquebrainz.data.model.user import User
+from custom_config import ADMINS
+
+moderators_bp = Blueprint('moderators', __name__)
+
+
+@moderators_bp.route('/moderators')
+def page():
+    users = User.query.filter(User.display_name.in_(ADMINS)).all()
```

Comment:
Database queries shouldn't be built in modules related to views. I know that we still do that in a lot of places, but that was a mistake which makes some things confusing. Apart from this, it would be better not to write any ORM code since we are trying to migrate away from that too.

There is now a new package `critiquebrainz.db` where database access code should go.